lib/fog/azurerm/models/storage/container.rb

```diff
@@ -0,0 +1,50 @@
+module Fog
+  module Storage
+    class AzureRM
+      # This class is giving implementation of create and delete a container.
+      class Container < Fog::Model
+        identity  :name
+        attribute :etag
+        attribute :last_modified
+        attribute :lease_duration
+        attribute :lease_state
+        attribute :lease_status
+        attribute :metadata
+        attribute :public_access_level
+
+        def create(options = {})
+          requires :name
+          merge_attributes(Container.flatten_properties(service.create_container(name, options)))
+        end
+
+        alias save create
+
+        def get_properties(options = { metadata: true })
+          requires :name
+          merge_attributes(Container.flatten_properties(service.get_container_properties(name, options)))
+        end
+
+        def get_access_control_list(options = {})
+          requires :name
+          merge_attributes(service.get_container_access_control_list(name, options)[0])
+        end
+
+        def delete(options = {})
+          requires :name
+          service.delete_container name, options
+        end
+
+        def self.flatten_properties(container)
```

Comment:
Please change this function name to 'parse' instead of 'flatten_properties'. You may have seen that we have followed that naming convention in all the models.